demo/secret/file/client.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"mosn.io/layotto/sdk/go-sdk/client"
+	runtimev1pb "mosn.io/layotto/spec/proto/runtime/v1"
+)
+
+func main() {
```

Comment:
We are trying to use one demo client to test all the components, see #455 
So can u remove redundent clients and just keep one ? 
You can take [the sequencer api quickstart](https://mosn.io/layotto/#/zh/start/sequencer/start) and [sequencer API demo client](https://github.com/mosn/layotto/blob/main/demo/sequencer/common/client.go#L19) as examples.
The storename can be passed to the client via `-s`. For example:
![image](https://user-images.githubusercontent.com/26001097/166858616-5d1a22cd-aac9-4f10-9a62-9fd7e232cb37.png)

Reply:
done